src/main/java/org/wildfly/security/_private/ElytronMessages.java

```diff
@@ -351,8 +351,8 @@
     @Message(id = 1077, value = "Invalid alias \"%s\" for missing mechanism database entry \"%s\"")
     void warnInvalidAliasForMissingMechanismDatabaseEntry(String value, String name);
 
-    @Message(id = 1078, value = "Ldap-backed realm failed to obtain identity from server")
-    RuntimeException ldapRealmFailedObtainIdentityFromServer(@Cause Throwable cause);
+    @Message(id = 1078, value = "Ldap-backed realm failed to obtain identity \"%s\" from server")
```

Comment:
Doesn't need changing in this PR but a couple of things I think we could clean up here: -

- Some of these messages could be made more generic with the realm type being passed in.
- We should get consistent re to use single or double quotes within messages - personally I tend to stick to single as it removes the need for escaping.